x/twap/migrate.go

```diff
@@ -14,3 +32,46 @@ func (k Keeper) MigrateExistingPools(ctx sdk.Context, latestPoolId uint64) error
 	}
 	return nil
 }
+
+// initializeGeometricTwapAcc initializes all existing twap records in state
+// with the given value. Returns nil on success, error otherwise.
+// Returns error if:
+// - fails to get historical time indexed records.
+// - does not find any time indexed records.
+// - the time indexed records returned from state are written back
+// in incorrect order. The order must be ascending by time so that
+// we update the "most recent" index with the right records correctly.
+func (k Keeper) initializeGeometricTwapAcc(ctx sdk.Context, value sdk.Dec) error {
```

Comment:
I think the current tests test this method indirectly by manually calling `updateGeometricAccum`, I think it would be better if we can test this method directly. Wdyt?